docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -0,0 +1,524 @@
+
+- Backing up your UWP project 
+- Convert project file (csproj) to SDK style
+- Clean up NuGet package references 
+- Update the target framework and NuGet packages 
+- Update package.appxmanifest
+- Removal of properties and items that are no longer required
+- Add template files such as App.Xaml, MainWindow.Xaml and publish profiles with options to build packaged and unpackaged app
+- Make namespace changes and add MainPage navigation
+- An attempt to detect and fix apis that have changed from UWP to Windows App SDK
+- An attempt to detect and mark apis that are either no longer supported or cannot be automatically converted with TODO comments and CLI messages
+
+We aim to provide migration guidance in form of warning messages within the tool and TODO comments within your project as the tool tries to migrate the project. In this way, you’ll always be in control of your migration. And for the APIs where complete automation is not possible, plan is to add //TODO comments for the developers to know where the work will be needed. A typical //TODO comment will also include a link to our existing migration documentation. Please check the Task list within the Visual Studio to see all the action items as TODO comments.
+
+## Things to know before starting your migration journey
+
+This tool currently supports C# and in most cases the app will require additional effort to complete the migration. The goal of the tool will be to convert your project and code so that it can compile, but features that require additional developer investigation will have TODO comments. Please check out the what’s supported page [here] (https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/what-is-supported)] before you begin your migration journey.
+
+Additionally, you may choose to wait for the next version of .Net UA tool before you start migrating your app because of the current limitations of the tool:
+
+- ApplicationView APIs not supported
+- AppWindow related APIs not supported (though it tries to generate a warning where possible and deliberately breaks your code so it doesn't compile till you manually fix things)
+- Custom Views supported (For example a CustomDialog that extends MessageDialog and calls an api incorrectly, it will not be warned about or fixed)
+- WinRT Components not supported
+- Multi window apps might not convert correctly
+- Apps that follow non standard file structure (App.xaml, App.xaml.cs not in root folder etc) might not be converted correctly
+
+Please note that this is currently a prerelease project and is receiving frequent updates. If you discover problems using the tool, report them in the tool's [GitHub repository](https://github.com/dotnet/upgrade-assistant). Please use the area tag as “UWP” so that all UWP related issues can be redirected to us.
```

Comment:
Again, quotes fix here, and I don't think public users can tag areas, can they? Double check on that.
```suggestion
Please note that this is currently a prerelease project and is receiving frequent updates. If you discover problems using the tool, report them in the tool's [GitHub repository](https://github.com/dotnet/upgrade-assistant). Please use the area tag as "UWP" so that all UWP related issues can be redirected to us.
```